src/Sql/Vault/dbo/Functions/UserCipherDetails_V2.sql

```diff
@@ -13,8 +13,16 @@ WITH [CTE] AS (
 )
 SELECT
     C.*,
-    COALESCE(CU.[ReadOnly], CG.[ReadOnly], 0) AS [Edit],
-    COALESCE(CU.[HidePasswords], CG.[HidePasswords], 0) AS [ViewPassword],
+    CASE
+        WHEN COALESCE(CU.[ReadOnly], CG.[ReadOnly], 0) = 0
+        THEN 1
+        ELSE 0
+    END [Edit],
+    CASE
+    	WHEN COALESCE(CU.[HidePasswords], CG.[HidePasswords], 0) = 0
+    	THEN 1
+    	ELSE 0
+	END [ViewPassword],
```

Comment:
⛏️ Drop one tab here 😜

Reply:
haha I guess Rider does not like to play well with SQL and tabs 🤦 

Fixed.